txlib/http/http_requests.py

```diff
@@ -153,18 +154,20 @@ def _send_json(self, method, path, data):
         headers = {'Content-type': 'application/json'}
         return self._make_request(method, path, data=data, headers=headers)
 
-    def _send_file(self, method, path, data, filename):
+    def _send_file(self, method, path, data, file):
         """Make a multipart/form-encoded request.
 
+        The file to send is opened in binary mode to report correct byte length.
+
         Args:
             `method`: The method of the request (POST or PUT).
             `path`: The path to the resource.
             `data`: The JSON-encoded data.
-            `filename`: The filename of the file to send.
+            `file`: The bytes (binary form) of the file to send.
```

Comment:
I believe that `file` is not a good name for this variable. I suggest `content` instead.